### DISCUSSION TITLE
How do I clear the rtl_433 data

### DISCUSSION THREAD

**Discussion by Markeydoo** [merbanan/rtl_433#2792] [General]
So it is a New UYear and I want to zero out my rain fall totals from my rtl_433 conected to my weather station.
I reset the weather station (Accurite 5 in 1) and it zeroed out byt my rtl_433 shows old data.
How can I clear it out?
Thanks

**Comment by zuckschwerdt** [comment]
You would actually need to reset the weather station, not the display unit. rtl_433 just displays the value and it is a "lifetime" value from the sensor.
